tools/bpf2c/bpf_code_generator.cpp

```diff
@@ -1233,7 +1233,8 @@ bpf_code_generator::bpf_code_generator_program::encode_instructions(
                     throw bpf_code_generator_exception(
                         "Map " + output.relocation + " doesn't exist", output.instruction_offset);
                 }
-                source = std::format("_maps[{}].address", std::to_string(map_definition->second.index));
+                source =
+                    std::format("runtime_context->map_data[{}].address", std::to_string(map_definition->second.index));
```

Comment:
Since generated code is changing, please reflect that in the `NativeCodeGeneration.md`.

Reply:
Updated.